### ISSUE TITLE
Improvement into code base

### ISSUE THREAD

**Issue by Khushalsarode** [rahulsainlll/git-trace#25]
- Raise issue template
- Raise PR template
- adding link to footer bar for redirect to repository to particular action
![Screenshot 2024-10-02 205130](https://github.com/user-attachments/assets/d9f351d0-6146-498a-85f5-fcb2a67fe0a1)
 - also dark mode can be added!

**Comment by GSingh2432002** [comment]
I wanted to add dark mode and light mode both for the website.
I want to work on this issue.
Under: Hacktoberfest

**Comment by rahulsainlll** [comment]
go ahead

**Comment by adityapandey23** [comment]
Can I work on issues other than the dark mode? @rahulsainlll

**Comment by rahulsainlll** [comment]
> I wanted to add dark mode and light mode both for the website. I want to work on this issue. Under: Hacktoberfest

you only gonna work on ? dark-mode of the complete issue

**Comment by GSingh2432002** [comment]
Yeah if you want to assign someone else with me, then we both can work on it.

**Comment by irohandev** [comment]
@rahulsainlll intrested for this...I can join @GSingh2432002 while doing it so.

**Comment by GSingh2432002** [comment]
@irohandev and @GSingh2432002 we both want to work on this issue. Kindly assign to @irohandev

**Comment by rahulsainlll** [comment]
yeah done

**Comment by Khushalsarode** [comment]
I was looking forward to work on issue?
can you assign?

**Comment by rahulsainlll** [comment]
> Yeah if you want to assign someone else with me, then we both can work on it.

yeh done

**Comment by rahulsainlll** [comment]
> I was looking forward to work on issue? can you assign? @rahulsainlll

there are already 3 ppls cant assign more to this issue 3 are more than enough

**Comment by adityapandey23** [comment]
@Khushalsarode 
where do you want me to link the report a bug and request a feature...both to the issues page ?
am I missing something?

**Comment by adityapandey23** [comment]
@Khushalsarode  @rahulsainlll is this fine ?
Should create a pr ?

**Comment by rahulsainlll** [comment]
yeah go ahead

**Comment by adityapandey23** [comment]
@rahulsainlll done.. you can check!
